pkg/cmd/app.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"github.com/NVIDIA/dcgm-exporter/pkg/dcgmexporter"
 	"github.com/NVIDIA/go-dcgm/pkg/dcgm"
+	"github.com/davecgh/go-spew/spew"
```

Comment:
It looks like go-spew has been abandoned and has not been maintained since 2018, do you think it might be better to use some other alternative?

Reply:
I agree, at the same time, Spew is a very popular package; for example, K8S uses it.

Reply:
I removed spew usage.

Reply:
Thank you!